modules/mobile/app/controllers/mobile/v0/payment_information_controller.rb

```diff
@@ -43,8 +58,35 @@ def ppiu_params
         )
       end
 
+      def lighthouse_ppiu_params
+        params[:routing_number] = params[:financial_institution_routing_number]
+        params.permit(:account_type,
+                      :account_number,
+                      :routing_number)
+      end
+
       def pay_info
-        @pay_info ||= EVSS::PPIU::PaymentAccount.new(ppiu_params)
+        @pay_info ||= if lighthouse?
+                        Lighthouse::DirectDeposit::PaymentAccount.new(lighthouse_ppiu_params)
+                      else
+                        EVSS::PPIU::PaymentAccount.new(evss_ppiu_params)
+                      end
+      end
+
+      def lighthouse_service
+        @client ||= DirectDeposit::Client.new(@current_user.icn)
+      end
+
+      def payment_account
+        @payment_account ||= Lighthouse::DirectDeposit::PaymentAccount.new(payment_account_params)
+      end
+
+      def lighthouse_adapter
+        Mobile::V0::Adapters::LighthouseDirectDeposit.new
+      end
+
+      def lighthouse?
+        Flipper.enabled?(:mobile_lighthouse_direct_deposit)
```

Comment:
Should we manage this per user or is the plan to flip it on for all users in staging?

Reply:
I'm not sure what the intent is on flipping. I haven't seen flippers used per person so I didn't know that was a consideration

Reply:
In most places we've been doing this feature flags per user like ` Flipper.enabled?(:mobile_lighthouse_direct_deposit, @current_user)`

In general I think this is probably the better way to do it because it allows us to not flip the entire flag on and off while testing.

Reply:
Updated. Thanks for catching that